psl/diagnostics/src/warning.rs

```diff
@@ -25,6 +25,11 @@ impl DatamodelWarning {
         Self::new(message, span)
     }
 
+    pub fn new_missing_index_on_emulated_relation(span: Span) -> DatamodelWarning {
+        let message = "The fields referenced in a `@relation` should be indexed to prevent potential performance penalties. Learn more at https://pris.ly/d/relation-mode-prisma-indexes.".to_owned();
```

Comment:
First suggestion, to fix the link: for the link:
```suggestion
        let message = "The fields referenced in a `@relation` should be indexed to prevent potential performance penalties. Learn more at https://pris.ly/d/relation-mode#indexes".to_owned();
```
The `#indexes` anchor will be added via this PR: https://github.com/prisma/docs/pull/4118